### PR TITLE
Fix for #Pydev 1127: Integrate flake8 in Pydev

### PR DESCRIPTION
**Pull request by oluiscabral**

### CODE REVIEW

**Comment by wesleybl** [comment]
Flake8 has an option to ignore files:

https://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations

It would be interesting for pydev to consider this setting and ignore the files. This would solve:

https://www.brainwy.com/tracker/PyDev/1124

**Comment by fabioz** [comment]
> Flake8 has an option to ignore files:
> 
> https://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations
> 
> It would be interesting for pydev to consider this setting and ignore the files. This would solve:
> 
> https://www.brainwy.com/tracker/PyDev/1124

Well, right now PyDev should run flake8 based on the project location, so, you should be able to configure the flake8 behaviour, but for ignoring code analysis in PyDev itself, we'd need a setting inside of PyDev, not really from an external tool.

**Comment by fabioz** [comment]
Note: icon should be something as: https://www.stockio.com/free-icon/nature-icons-snowflake

File: plugins/org.python.pydev.shared_core/src/org/python/pydev/shared_core/IMiscConstants.java [hunk 1]
```diff
@@ -33,6 +33,9 @@
     String MYPY_PROBLEM_MARKER = "org.python.pydev.mypyproblemmarker";
     String MYPY_MESSAGE_ID = "mypy_message_id";
 
+    String FLAKE8_PROBLEM_MARKER = "org.python.pydev.flake8problemmaker";
```
Comment (fabioz) [inline plugins/org.python.pydev.shared_core/src/org/python/pydev/shared_core/IMiscConstants.java:36]: Note: you also need to register the `flake8problemmaker` in the `org.python.pydev/plugin.xml` (search for `mypyproblemmarker`) in that file.

**Review by fabioz — CHANGES_REQUESTED** [review]

**Comment by fabioz** [comment]
This was integrated in: https://github.com/fabioz/Pydev/commit/59140d0abb17230b3c7e1ac31da8463fba558607
Some additional fixes were done in: https://github.com/fabioz/Pydev/commit/6f7fd48309d3d3951d76f1ad2671c458fae28063

**Pull request closed without merging**